manifests/u/Ultimaker/Cura/4.10.0/Ultimaker.Cura.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Ultimaker.Cura
```

Comment:
Cura can't be added as it asks to install drivers during install. If the user doesn't trust Ultimaker's driver signing certificate, Windows asks them if they want to install the drivers, which can't be suppressed. 

See #23391 and others for more info.

Reply:
If they don’t work. Not all versions of Cura try to install drivers at install time.

Reply:
The other versions should stay. This started with 4.10.

Ultimaker/Cura#10091
Ultimaker/Cura#1245
Ultimaker/Cura/issues/8439

Be sure to 👍 the open Issues there.